app/utils/cyaHelpers/SummaryListRowHelper.scala

```diff
@@ -51,7 +52,7 @@ private[utils] class SummaryListRowHelper(implicit messages: Messages) {
         format.setCurrency(currency)
         format.format(answer)
       } catch {
-        case _: Throwable => s"$answer ${currencyCode.currency}"
+        case NonFatal(_: Throwable) => s"$answer ${currencyCode.currency}"
```

Comment:
I wonder if using `Try` with a `.getOrElse` might be a better scala approach here? Also might be worth adding some tests for this method as I can't see any 😓